schema/spacewalk/upgrade/susemanager-schema-4.2.2-to-susemanager-schema-4.2.3/001-rhnActionDup-add-alllow-vendor-change.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE rhnActionDup DROP CONSTRAINT IF EXISTS rhn_actiondup_avc_ck;
+ALTER TABLE rhnActionDup
```

Comment:
You should have this twice. It would be good to keep the 4.1 version as  it is. This is needed for migration.